R/RadeonProRender/build_tarballs.jl

```diff
@@ -3,19 +3,19 @@
 using BinaryBuilder, Pkg
 
 name = "RadeonProRender"
-version = v"3.1.00"
+version = v"3.1.2"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK.git", "f880013a9b263c7ec5143d3f9246fe8fb58959ba")
+    GitSource("https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK.git", "20d32d8c27269cd407190d98c7c3b63d150d3c47")
 ]
 
 # TODO, also ship headers for Clang.jl generation!?
 script = raw"""
 echo ${target}
 cd $WORKSPACE/srcdir/RadeonProRenderSDK/RadeonProRender
 if [[ ${target} == x86_64-linux-gnu ]]; then
-    cp binUbuntu18/* ${libdir}/
+    cp binUbuntu20/* ${libdir}/
```

Comment:
Please use verbose commands, so we can see what's going on
```suggestion
    cp -v binUbuntu*/* ${libdir}/
```
Maybe also below